common/get_system_info.yml

```diff
@@ -11,7 +11,7 @@
 - name: "Filter is not specified"
   block:
     - name: "Collect guest information for '{{ vm_guest_ip }}'"
-      ansible.builtin.setup:
+      ansible.windows.setup:
```

Comment:
This one can keep using `ansible.builtin.setup` otherwise Linux guest OS won't work.

Reply:
OK, I'll test Linux guest OS.